docs/section5/Rule5-7-2.md

```diff
@@ -32,4 +32,4 @@
             - Repeat previous steps to calculate the User WWR for each orientation and building area type: `_WWR_user = _calculate_wwr(U_RMR)`  
             - **Rule Assertion** For each building area type and orientation the WWR is equal between the Baseline and User RMRs: `_WWR_baseline_ == _WWR_user`
```

Comment:
for space conditioning category, we should use the heating and cooling capacity, as clarified in 5-2
This check is to verify that the windows are distributes in the correct proportions- please see example calculations in the envelope TCD spreadsheet, tab 'Calcs_Rule5.7'. Step 2 calculates the window proportions in the U-RMR and Step 4 does the same for the B-RMR.